sdk/storage/storage-file-datalake/src/models.ts

```diff
@@ -1221,6 +1222,18 @@ export interface FileAppendOptions extends CommonOptions {
    * If file should be flushed automatically after the append
    */
   flush?: boolean;
+  /**
+   * Proposed lease ID, in a GUID string format. The Blob service returns 400 (Invalid request) if the proposed lease ID is not in the correct format. See Guid Constructor (String) for a list of valid GUID string formats.
+   * */
+  proposedLeaseId?: string;
+  /**
+   * The lease duration is required to acquire a lease, and specifies the duration of the lease in seconds.  The lease duration must be between 15 and 60 seconds or -1 for infinite lease.
+   * */
+  leaseDuration?: number;
```

Comment:
I wonder why we didn't have a unit in the name of the existing `PathCreateOptions.leaseDuration`.  Usually we would prefer `leaseDurationInSeconds`.

Reply:
I'll still keep naming consistent with the old ones.  But, yes, I also think a unit in name is a better way to let customer know about its unit.